extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -69,11 +69,26 @@ oppia.directive('imageWithRegionsEditor', [
         $scope.rectY = 0;
         $scope.rectWidth = 0;
         $scope.rectHeight = 0;
-
         // Is user currently drawing a new region?
         $scope.userIsCurrentlyDrawing = false;
         // Is user currently dragging an existing region?
         $scope.userIsCurrentlyDragging = false;
+        // Is user currently resizing an existing region?
+        $scope.userIsCurrentlyResizing = false;
+        // The horizontal direction along which user resize occurs
+        // 1 -> Left -1 -> Right 0 -> No resize
```

Comment:
Here and below: can you put 3-4 more spaces after "Left" and after "Right"? Otherwise it's all too easy to read it as "Left -1" or "Right 0".